app/assets/javascripts/components/StickyColumn.js

```diff
@@ -0,0 +1,215 @@
+define(['jquery', 'DoughBaseComponent', 'eventsWithPromises'], function($, DoughBaseComponent, eventsWithPromises) {
+  'use strict';
+
+  var StickyColumn,
+      defaultConfig = {},
+      $parent,
+      contentHeight,
+      topMargin,
+      top,
+      bottom,
+      isFixed,
+      isAtBottom,
+      isInSidebar,
+      mainContentSelector = '.l-article-3col-main',
```

Comment:
These selectors could be moved into `defaultConfig`: 
```javascript
defaultConfig: { 
   selectors: { 
     mainContentSelector: '.l-article-3col-main'
     // ...
   }
}
``` 

as it will allow them to be set override by the element config `<div data-dough-sticky-column-config="{'mainContentSelector':'.my-custom-class'}".`